analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -337,7 +337,7 @@ data class GemSpec(
             )
         }
 
-        private val GITHUB_REGEX = Regex("https*:\\/\\/github.com\\/(?<owner>[\\w-]+)\\/(?<repo>[\\w-]+)")
+        private val GITHUB_REGEX = Regex("https*:\\/\\/github.com\\/(?<owner>[\\w-]+)\\/(?<repo>[\\w-\\.]+)")
```

Comment:
How about further simplifying this and allow everything *except* `/`? Like:

    Regex("https?:\\/\\/github.com\\/(?<owner>[^\\/]+)\\/(?<repo>[^\\/]+)")

While at it, I also changed `s*` to the more speaking `s?` in the beginning.

Reply:
Right, this should be fine since we are not trying to validate GitHub URLs. In the worst case we will need a curation, same as if VCS info is missing. Fixed.